rundeck-data-model/src/main/java/org/rundeck/app/data/providers/v1/WebhookDataProvider.java

```diff
@@ -0,0 +1,37 @@
+package org.rundeck.app.data.providers.v1;
+
+import org.rundeck.app.data.model.v1.webhook.RdWebhook;
+import org.rundeck.app.data.model.v1.webhook.dto.SaveWebhookRequest;
+import org.rundeck.app.data.model.v1.webhook.dto.SaveWebhookResponse;
+
+import java.util.List;
+
+public interface WebhookDataProvider extends DataProvider {
+    RdWebhook get(Long id);
+    RdWebhook getWebhook(Long id);
```

Comment:
`get(Long)` and `getWebhook(Long)` do the same thing, should be combined into one.